Aydin/frontend/dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:13
+
+WORKDIR /home/aydin
+ENV PATH ="${PATH}:/home/aydin/node_modules/.bin/"
+
+COPY package.json ./
+
```

Comment:
do people normally use docker for react apps? i haven't heard of this actually, this doesn't seem normal to me. but if you've looked it up then ok

Reply:
actually yes, its kinda commen to use dicker for react apps